samples/javascript_nodejs/23.facebook-events/bots/facebookBot.js

```diff
@@ -0,0 +1,199 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { ActivityHandler, TurnContext, CardFactory , ActionTypes} = require('botbuilder');
+const { ChoiceFactory } = require('botbuilder-dialogs');
+// These are the options provided to the user when they message the bot
+const facebookPageIdOption = "Facebook Page Id";
+const quickRepliesOption = "Quick Replies";
+const postBackOption = "PostBack";
+
+class FacebookBot extends ActivityHandler {
+
+    /**
+     * @param {any} logger object for logging events, defaults to console if none is provided 
+     */
+    constructor(logger) {
+        super();
+        if (!logger) {
+            logger = console;
+            logger.log('[FacebookEventsBot]: logger not passed in, defaulting to console');
+        }
+
```

Comment:
please remove the extra line of whitespace

Reply:
removed